src/main/java/engine/ItemManager.java

```diff
@@ -28,11 +28,13 @@ public class ItemManager {
     /** Height of game screen. */
     private int HEIGHT;
     /** Item drop probability, (1 ~ 100). */
-    private static final int ITEM_DROP_PROBABILITY = 30;
+    private static final int ITEM_DROP_PROBABILITY = 100;
```

Comment:
It looks like it should be reverted back to 30%.

```suggestion
    private static final int ITEM_DROP_PROBABILITY = 30;
```